txpool/txpool.go

```diff
@@ -184,12 +208,16 @@ func (t *TxPool) AddTx(tx *types.Transaction) error {
 
 // addImpl validates the tx and adds it to the appropriate account transaction queue.
 // Additionally, it updates the global valid transactions queue
-func (t *TxPool) addImpl(ctx string, tx *types.Transaction) error {
+func (t *TxPool) addImpl(ctx TxOrigin, tx *types.Transaction) error {
```

Comment:
I'd suggest to rename the `ctx` to something like `origin` to avoid confusion with the context package

Reply:
Fixed in https://github.com/0xPolygon/polygon-sdk/pull/151/commits/d5e92b6f828f644bfbf9e025fc62d6bea106cf42